Makefile.uk

```diff
@@ -0,0 +1,153 @@
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcinit.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcmainct.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcmarker.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcmaster.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcomapi.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcparam.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcprepct.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jcsample.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jctrans.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdapimin.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdapistd.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdarith.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdatadst.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdatasrc.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdcoefct.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdcolor.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jddctmgr.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdhuff.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdinput.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdmainct.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdmarker.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdmaster.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdmerge.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdpostct.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdsample.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jdtrans.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jerror.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jfdctflt.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jfdctfst.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jfdctint.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jidctflt.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jidctfst.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jidctint.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jquant1.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jquant2.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jutils.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jmemmgr.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/jmemnobs.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdppm.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdgif.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdtarga.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdrle.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdbmp.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdswitch.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/cdjpeg.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/wrppm.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/wrgif.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/wrtarga.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/wrrle.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/wrbmp.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/rdcolmap.c
+LIBJPEG_SRCS-y += $(LIBJPEG_EXTRACTED)/transupp.c
+
+################################################################################
+# JPEG prepare
+################################################################################
+# Run ./configure
+$(LIBJPEG_EXTRACTED)/config.status: $(LIBJPEG_BUILD)/.origin
+	$(call verbose_cmd,CONFIG,libjpeg: $(notdir $@), \
+		cd $(LIBJPEG_EXTRACTED) && ./configure)
```

Comment:
This step needs to be removed.  The `config.status` file needs to be explicitly created and placed within a to-level `include/` directory.  The directory and corresponding header file should be referenced via `LIBJPEG_COMMON_INCLUDES`.  It's options need be checked and must correspond with what is compatible with Unikraft.